CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## Wazuh Chef v4.0.4_1.11.0
+
+### Added
+- Update to Wazuh ([v4.0.4](https://github.com/wazuh/wazuh/blob/v4.0.4/CHANGELOG.md#v404))
```

Comment:
This will render the version inside parenthesis. Remove the outer ones to match the format used on other repos.